.github/workflows/deploy.yml

```diff
@@ -53,12 +46,57 @@ jobs:
             cd go-tableland
             git fetch && git checkout -f ${{ github.sha }}
             cd docker
-            
-            HEALTHBOT_ENABLED=true make ${{ steps.prep.outputs.deployment }}-up
+
+            HEALTHBOT_ENABLED=true make staging-up
 
             # Wait to start and double-check we're running the expected version.
             # This helps having a green check in the GH Action be high-signal that everything is fine.
             sleep 5;
             for i in 1 2 3 4; do [ $(curl --insecure --silent https://127.0.0.1/version | jq .git_commit) = "\"${{ steps.prep.outputs.sha_short}}\"" ] && break || (if [ $i = 4 ]; then exit -1; else sleep 5; fi ) done 
             echo "All healthy!"
 
+      - if: github.ref == 'refs/heads/main'|| contains(github.event.head_commit.message, '[testnetdeploy]')
+        name: Deploy testnet
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.TESTNET_V2_VM_HOST }}
```

Comment:
I added this `TESTNET_V2_VM_HOST` to avoid conflicts with the current secret.
After we prune the old testnet validator, we can rename this secret again.